lib/pbench/server/database/models/users.py

```diff
@@ -66,6 +72,12 @@ def add(self):
             Database.db_session.rollback()
             raise
 
+    @validates("role")
+    def evaluate_role(self, key, value):
+        if value.lower() in ["admin", "administrator"]:
+            return Roles(1)
```

Comment:
It'd be better to use `Roles[value.upper()]` to convert the string directly; and, again, I really don't like having two separate keywords here to translate to one role. Let's just stick with "admin" which maps to Roles.ADMIN. (Enums should generally be all-caps.)